features/user_can_open_post.feature

```diff
@@ -13,16 +13,15 @@ Feature: Open up posts
     | Codeacademy  | Smooth-ride   | Fat Bob  |
     | Hacklife     | smaaaahht     | Slim Jim |
     And I am logged in as "Fat Bob"  
-   
+
   Scenario: Reader can open up posts
     When I visit the site
-    And I click "FORUM"
+    And I click "FLORUM"
+    Then I should see "MARKETPLACE"
```

Comment:
Why add this assertion here?
You are not testing the categories here

Reply:
Removed

Reply:
This test doesn't work now. We used binding pry and everything shows up, but the test still fails :\